src/System.ServiceModel.Duplex/tests/System.ServiceModel.Duplex.Tests.builds

```diff
@@ -6,6 +6,10 @@
       <TargetGroup>netstandard1.3</TargetGroup>
       <TestTFMs>netcore50;netcoreapp1.1</TestTFMs>
```

Comment:
You've added the 2.0/2.0 configuration to all the builds files, which is fine.  The thing I want to make sure we cover is the ns1.3/nca2.0 scenario.